packages/react-ui-core/src/List/List.js

```diff
@@ -70,7 +70,13 @@ export default class List extends PureComponent {
         )}
       >
         {items.map((item, i) => (
-          <Item key={this.itemId(i)} nodeType={listItemNodeType} {...props}>
+          <Item
+            key={this.itemId(i)}
+            index={i}
+            nodeType={listItemNodeType}
+            theme={theme}
```

Comment:
You shouldn't be passing `theme` here. It gets passed down by the theme provider through context.